pkg/vault/contracts/authorizer/Executor.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/Address.sol";
+
+import "./TimelockAuthorizer.sol";
+
+contract Executor {
```

Comment:
I'd rename this to something like TimelockExecutor, or ScheduledActionExecutor (I quite like this second one)

Reply:
I named it exactly `TimelockExecutor` initially but I ended up changing it to simply `Executor` since there is nothing related to timelocks in this contract. Let me know your thoughts.

Reply:
There is a lot about timelocks: only the authorizer can call this, which it does because of timelocks :stuck_out_tongue: